packages/ui-text-input/README.md

```diff
@@ -4,8 +4,6 @@ category: packages
 
 ## ui-text-input
 
-[![npm][npm]][npm-url]&nbsp;
```

Comment:
Here you deleted the npm link by accident

Reply:
omg, let me check again all changes

Reply:
I will fix the mistake and push again.